packages/shared/lib/clients/orchestrator.ts

```diff
@@ -173,23 +173,7 @@ export class Orchestrator {
             }
 
             if (res.isErr()) {
-                await createActivityLogMessageAndEnd({
-                    level: 'error',
-                    environment_id,
-                    activity_log_id: activityLogId,
-                    timestamp: Date.now(),
-                    content: `Failed with error ${res.error.type} ${JSON.stringify(res.error.payload)}`
-                });
-                await logCtx.error(`Failed with error ${res.error.type}`, { payload: res.error.payload });
-                await createActivityLogMessageAndEnd({
-                    level: 'error',
-                    environment_id,
-                    activity_log_id: activityLogId,
-                    timestamp: Date.now(),
-                    content: `The action workflow ${workflowId} did not complete successfully`
-                });
-                await logCtx.error(`The action workflow ${workflowId} did not complete successfully`);
-                return res;
+                throw res.error;
```

Comment:
logging would be taken care by the try catch https://github.com/NangoHQ/nango/pull/2278/files#diff-485470c374a0419d6a217c25b8d8dcee97ad5959e068630d075b77a1d7bc423fL222-L234